<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>401</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Stock Trading Bot&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextDayButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>100</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Day</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseStocksButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>View/Select Stocks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bankButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>220</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="dayLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Day: 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="consoleLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>211</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
    &lt;p align=&quot;center&quot; style=&quot;color: black;&quot;&gt;Hello, I will trade stocks for you&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="speechLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="doneButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>100</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>nextDayButton</zorder>
     <zorder>chooseStocksButton</zorder>
     <zorder>imageLabel</zorder>
     <zorder>dayLabel</zorder>
     <zorder>speechLabel</zorder>
     <zorder>consoleLabel</zorder>
     <zorder>bankButton</zorder>
     <zorder>doneButton</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="decisionLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>311</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Hello, please select&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;your Bot's strategy! &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dailyButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Make a decision DAILY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="weeklyButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Make a decision WEEKLY</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Stocks&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goBackButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>190</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stockLabel8">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="regalView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gamestopView">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nvidiaView">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="teslaView">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>110</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nokiaView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="microsoftView">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="amazonView">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="appleView">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>280</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="goBackButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="bankLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Amount: $1000&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="bankLabel_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;BANK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tenButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>$10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="twentyButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>$20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fiftyButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>$50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hundredButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>270</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>$100</string>
      </property>
     </widget>
     <widget class="QLabel" name="bankLabel_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Deposit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="bankLabel_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;BOT Summary&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="summaryLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>521</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;lol&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
